bolt-simple-taproot.md

```diff
@@ -257,86 +255,64 @@ before aggregating (`KeyAgg(KeySort(p1, p2))`).
 
 #### Nonce Generation
 
-A `musig2` public nonce is a 66-byte value which is the concentration of two EC
-points encoded using the compressed format:
+A `musig2` secret nonce is the concatenation of two random, 32-byte integers.
```

Comment:
33-byte?

Reply:
No, the secret nonce components are 32 bytes each.